includes/subscriptions/templates/html-wcpay-deactivate-warning.php

```diff
@@ -20,32 +20,42 @@
 					<p>
 						<?php
 							printf(
-								// translators: $1 $2 $3 placeholders are opening and closing HTML link tags, linking to documentation. $4 $5 placeholders are opening and closing strong HTML tags.
-								esc_html__( 'Your store has active subscriptions using the built-in WooCommerce Payments functionality. Due to the %1$soff-site billing engine%3$s these subscriptions use, %4$sthey will continue to renew even after you deactivate WooCommerce Payments%5$s. %2$sLearn more%3$s.', 'woocommerce-payments' ),
+							// translators: $1 $2 $3 placeholders are opening and closing HTML link tags, linking to documentation. $4 $5 placeholders are opening and closing strong HTML tags. $6 is WooPayments.
+								esc_html__( 'Your store has active subscriptions using the built-in %6$s functionality. Due to the %1$soff-site billing engine%3$s these subscriptions use, %4$sthey will continue to renew even after you deactivate WooCommerce Payments%5$s. %2$sLearn more%3$s.', 'woocommerce-payments' ),
```

Comment:
Forgot to replace one instance of "WooCommerce Payments" with `%6$s`

Reply:
amazing catch, thanks!
fixed in https://github.com/Automattic/woocommerce-payments/pull/6581/commits/b1d36c896aa92104e381f3d5d35fad7804264515